### ISSUE TITLE
Anti x-ray issues

### ISSUE THREAD

**Issue by L4BORG** [MylesIsCool/ViaVersion#201]
### What version of ViaVersion are you using? Type /ver viaversion
ViaVersion 0.6.3

### What version of Spigot are you using? Type /ver
CraftBukkit version git-PaperSpigot-"8b18730" (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT)

### What plugins are you using? Type /plugins
Plugins (32): ServerManager, AsyncWorldEditInjector, Deadbolt, WorldEdit, OpenInv, LogBlock, PlayerWarps, Buycraft, Chairs, WorldBorder, Votifier, SilkSpawners, WorldGuard, AutoSaveWorld, AsyncWorldEdit, GlobalMarket, EchoPet, ViaVersion, GAListener, Vault, GriefPrevention, ChestShop, CombatTagPlus, Essentials, TrailGUI, ProtocolLib, NoCheatPlus, TradeMe, PlayerVaults, UserManager, EssentialsSpawn, HolographicDisplays

### Are you using any additional software like BungeeCord? If so, what software and version? (Please list your plugins as well)
Latest bungeecord, ServerListPlus, SkinsRestorer, NuVotifier

### How does this error happen? login? Using an item? 
instantly after /timings on

### Is there an error in the console? Use pastebin.com. Is there a kick message?
```
[14:39:00 ERROR]: stopTiming called async for anti-xray - obfuscate
[14:39:00 WARN]: java.lang.Throwable
[14:39:00 WARN]:        at co.aikar.timings.TimingHandler.stopTiming(TimingHandler.java:114)
[14:39:00 WARN]:        at org.spigotmc.AntiXray.obfuscateSync(AntiXray.java:103)
[14:39:00 WARN]:        at net.minecraft.server.v1_8_R3.PacketPlayOutMapChunk.<init>(PacketPlayOutMapChunk.java:22)
[14:39:00 WARN]:        at sun.reflect.GeneratedConstructorAccessor1247.newInstance(Unknown Source)
[14:39:00 WARN]:        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[14:39:00 WARN]:        at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
[14:39:00 WARN]:        at us.myles.ViaVersion.handlers.ViaChunkHandler.encode(ViaChunkHandler.java:40)
[14:39:00 WARN]:        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:89)
[14:39:00 WARN]:        at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:658)
[14:39:00 WARN]:        at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:716)
[14:39:00 WARN]:        at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:706)
[14:39:00 WARN]:        at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:741)
[14:39:00 WARN]:        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:895)
[14:39:00 WARN]:        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:240)
[14:39:00 WARN]:        at com.comphenix.protocol.compat.netty.independent.NettyChannelProxy.writeAndFlush(NettyChannelProxy.java:327)
[14:39:00 WARN]:        at net.minecraft.server.v1_8_R3.NetworkManager$4.run(NetworkManager.java:198)
[14:39:00 WARN]:        at com.comphenix.protocol.compat.netty.independent.NettyChannelInjector$5$3.run(NettyChannelInjector.java:345)
[14:39:00 WARN]:        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:380)
[14:39:00 WARN]:        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:268)
[14:39:00 WARN]:        at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
[14:39:00 WARN]:        at java.lang.Thread.run(Thread.java:745)
```

Is this ViaVersion issue?

**Comment by lenis0012** [comment]
Yes.
Should be fixed by #200

**Comment by MylesIsCool** [comment]
Fixed in next version.
